### DISCUSSION TITLE
Two-sided option

### DISCUSSION THREAD

**Discussion by mkiatos** [joaomlourenco/novathesis#336] [Q&A] [answered]
Is there a twosided option?

**Comment by joaomlourenco** [comment, marked as answer]
Yes!  Select option "media = paper"

**Reply by mkiatos** [reply]
I removed the  the screen option from the cls file i.e. media/.new choice = {paper} and it still it does not work. Is there any other option?

**Reply by joaomlourenco** [reply]
Why did you do that?  Since when you are supposed to change the cls file??
Get the latest version and select the option "media=paper" in file `Config/1_novathesis.tex`.
